src/frontend/components/Chat/SharedChatComponents/InputBar/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, Button, Text, TextInput } from 'grommet';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { SendButtonSVG } from 'components/SVGIcons/SendButtonSVG';
+
+const ButtonStyled = styled(Button)`
+  padding: 0;
+`;
+
+interface InputBarProps {
+  handleUserInput: (inputText: string) => boolean;
+  isChatInput: boolean;
+  placeholderText: string;
+}
+
+export const InputBar = ({
+  handleUserInput,
+  isChatInput,
+  placeholderText,
+}: InputBarProps) => {
+  const [inputText, setInputText] = useState('');
+
+  const handleOnClick = () => {
+    if (inputText.length !== 0) {
+      if (handleUserInput(inputText)) {
+        setInputText('');
+      }
+    }
+  };
+
+  const handleKeypress = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.code === 'Enter') {
+      handleOnClick();
+    }
+  };
+
+  return (
+    <Box
+      align="center"
+      border={{
+        color: 'blue-chat',
+        size: 'xsmall',
+      }}
+      direction="row"
+      height={isChatInput ? '50px' : '35px'}
+      pad={{
+        right: isChatInput ? '5px' : '3px',
+      }}
+      round="6px"
+    >
+      <TextInput
+        focusIndicator={false}
+        onChange={(event) => setInputText(event.target.value)}
+        onKeyPress={(keyPressEvent) => handleKeypress(keyPressEvent)}
+        placeholder={
+          <Box style={{ display: 'inline-grid' }} width="100%">
```

Comment:
you should probably use style-component here and eventually create a subcomponent for this

Reply:
I'm afraid it could be a bit heavy for only one style lien IMO